sql/updates/auth/2015_02_xx_xx_auth.sql

```diff
@@ -0,0 +1,77 @@
+UPDATE `rbac_permissions` SET `name`='Command: reload support system' WHERE  `id`=650;
+
+DELETE FROM `rbac_permissions` WHERE `id` IN (799,800,801,802,803,804,805,806,807,808,809,810,811,812,813,814,815,816,817,818,819,820,821,822,823,824,825,826,827,828,829,830,831,832,833);
```

Comment:
new permissons were added. file has to be applyable multiple times, so delete is needed.